lib/DiscoveryStreamFeed.jsm

```diff
@@ -58,6 +58,7 @@ const PREF_IMPRESSION_ID = "browser.newtabpage.activity-stream.impressionId";
 const PREF_ENABLED = "discoverystream.enabled";
 const PREF_HARDCODED_BASIC_LAYOUT = "discoverystream.hardcoded-basic-layout";
 const PREF_SPOCS_ENDPOINT = "discoverystream.spocs-endpoint";
+const PREF_LANG_LAYOUT_CONFIG = "discoverystream.lang-layout-config";
```

Comment:
This pref needs to exist in a patch with MC which I have not created yet.

If you want to test it now, you can add this string pref:
`browser.newtabpage.activity-stream.discoverystream.lang-layout-config` and give the value of "en" or "en,de"